package.yaml

```diff
@@ -147,14 +147,16 @@ executables:
       - -rtsopts
       - -with-rtsopts=-N
     dependencies:
+      - aeson
+      - ansi-wl-pprint
+      - elliptic-curve
+      - galois-field
+      - gitrev
+      - haskeline
```

Comment:
Probably would be best if we comment what each section of deps are for, I forgot to do that the last time I was here